iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/checks/StorageLimitCheckTest.java

```diff
@@ -22,22 +22,32 @@
 import org.junit.jupiter.api.Test;
 import org.sonar.iac.common.api.checks.IacCheck;
 
-class EphemeralStorageLimitCheckTest {
-  IacCheck check = new EphemeralStorageLimitCheck();
+class StorageLimitCheckTest {
```

Comment:
Do we need tests with different namespaces here as well?

Reply:
E.g. `testKindWithTemplateAndNamespace()` & `testGlobalLimitRangeNoIssues()` verify different namespaces.